internal/model/transformations.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+// WtsToString transforms struct with sql.NullString values and returns formatted string
+// seems like should be replaced by custom marshaling/unmarshaling methods
+func (wtl WasteTypeList) WtsToString() string {
```

Comment:
Избыточное название. Ресивер метода уже называется `WasteTypeList`, понятно, что метод приведения к строке будет конвертировать именно этот тип.

```suggestion
func (wtl WasteTypeList) String() string {
```